massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -1279,12 +1257,11 @@ impl ConsensusWorker {
         for b_id in new_final_block_ids.into_iter() {
             if let Some(a_block) = self.block_db.get_active_block(&b_id) {
                 // List new final ops
-                new_final_ops.extend(
-                    a_block
-                        .operation_set
-                        .iter()
-                        .map(|(id, (_, exp))| (*id, (*exp, a_block.slot.thread))),
-                );
+                new_final_ops.extend(a_block.operation_set.iter().map(|(id, _)| {
```

Comment:
We use this vector to pass it to notify the pool just above (L.1276) but the operations are already here if you can fetch them with the storage

Reply:
Ok, thanks for the info, I'll look into it and will remove it if possible...

Reply:
I think the point is not to add the operations to storage, they are indeed already there, but rather to notify pool of them becoming final.